lib/wallet/wallet.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:ten_ten_one/menu.dart';
+import 'package:ten_ten_one/wallet/wallet_bitcoin.dart';
+import 'package:ten_ten_one/wallet/wallet_change_notifier.dart';
+import 'package:ten_ten_one/wallet/wallet_dashboard.dart';
+import 'package:ten_ten_one/wallet/wallet_lightning.dart';
+
+class Wallet extends StatefulWidget {
+  static const route = '/cfd-trading';
+
+  const Wallet({Key? key}) : super(key: key);
+
+  @override
+  State<Wallet> createState() => _WalletState();
+}
+
+class _WalletState extends State<Wallet> {
+  final List<Widget> _pages = <Widget>[
+    const WalletDashboard(),
+    const WalletLightning(),
+    const WalletBitcoin(),
+  ];
+
+  final List<String> _pageNames = <String>[
+    WalletDashboard.name,
+    WalletLightning.name,
+    WalletBitcoin.name,
+  ];
+
+  final List<String> _pageNavigationLabels = <String>[
+    WalletDashboard.navigationLabel,
+    WalletLightning.navigationLabel,
+    WalletBitcoin.navigationLabel,
+  ];
+
+  final List<IconData> _pageIcons = <IconData>[
+    WalletDashboard.icon,
+    WalletLightning.icon,
+    WalletBitcoin.icon,
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    WalletChangeNotifier walletChangeNotifier = context.watch<WalletChangeNotifier>();
+
+    return Scaffold(
+        appBar: AppBar(
```

Comment:
🔧 I guess it would be nicer if the appbar is not defined by wallet, but by the wallet lightning, bitcoin or dashboard widget. That way you don't have to maintain this list of page names.

Reply:
feel free to create follow up for this :)